portforwards.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
```

Comment:
nit: probably good to put this in the `hack` dir this conversion comes from K8s: https://github.com/kubernetes/kubernetes/tree/master/hack